setup.py

```diff
@@ -20,11 +21,14 @@ def fread(fname):
 
 if sys.version_info < (2, 7):
     required.append('ordereddict')
+if sys.version_info < (3,):
+    required.append('configparser')
```

Comment:
I just checked: support for the `fallback` keyowrd arg to `ConfigParser.get()` was introduced in Python 3.2, so that's the minimum version that works without the backported `configparser` module. I updated `setup.py` accordingly.